ipmininet/router/config/zebra.py

```diff
@@ -155,28 +178,99 @@ def __iter__(self):
         return iter(self._match)
 
 
+class RouteMapMatchCond(object):
+    """
+    A class representing a RouteMap matching condition
+    """
+
+    def __init__(self, type, condition):
```

Comment:
type is a built-in function in python so it would be better to choose another name for your variable